core/src/main/scala/org/apache/spark/sql/SnappySession.scala

```diff
@@ -1613,9 +1618,47 @@ object SnappySession extends Logging {
     }
   }
 
+  class CachedKey(val session: SnappySession, val lp: LogicalPlan, val sqlText: String, val hintHashcode: Int) {
```

Comment:
Good to see Soubhik and others being forced to be careful about formatting by Idea :-)